### ISSUE TITLE
Use default config file

### ISSUE THREAD

**Issue by WuglyakBolgoink** [BePo65/license-report#102]
it will be great to use default config file.

example:
![image](https://user-images.githubusercontent.com/2271337/180782609-9c642284-0b1b-40ed-8168-9c3cedc863b1.png)

And then anyone can change it, e.g.:
```json
{
  "fields": [
    "name",
    "licenseType",
    "author",
    "link",
    "installedVersion",
  ],
  "output": "html",
  "only": "prod,dev",
  "registry": "http://example.com"
}
```

currently I need to specify config file manually:

```json
    "build:license-report": "license-report --config=.license-report-config  > ./src/dependencies.html"
```

**Comment by BePo65** [comment]
Oh, i found out that this is a default feature of the `rc` package that we use to handle the configuration. So just create a file named `.license-reportrc` in the project root and fill it with the desired options (as json). When you run license-report in the directory of the projects package.json this file will be used for configuration.

**Comment by BePo65** [comment]
Solved by pr #106
